host.yaml

```diff
@@ -0,0 +1,89 @@
+ref: "openshift/3.10/x86_64/os"
+repos:
+  - atomic-centos-continuous
+  - rhcos-continuous
+  - openshift
+  - rhel-7.5-server
+  - rhel-7.5-server-optional
+  - rhel-7.5-server-extras
+  - rhel-7.5-atomic
+install_langs:
+ - en_US
+documentation: false
+initramfs-args:
+  - "--no-hostonly"
+  - "--add"
+  - "iscsi"
+automatic_version_prefix: "3.10-7.1803"
+mutate-os-release: "7"
+postprocess-script: "compose-post.sh"
+etc-group-members:
+  - wheel
+  - docker
+ignore-removed-users:
+  - root
+ignore-removed-groups:
+  - root
+check-passwd:
+  type: "file"
+  filename: "passwd"
+check-groups:
+  type: "file"
+  filename: "group"
+packages: 
+ # Kernel
+ - kernel irqbalance microcode_ctl
+ # rpm-ostree
+ - rpm-ostree nss-altfiles
+ # Subscriptions
+ - subscription-manager subscription-manager-plugin-ostree
+ - subscription-manager-plugin-container
+ # SELinux
+ - selinux-policy-targeted policycoreutils-python
+ - setools-console
+ # System setup
+ - ignition cloud-init cloud-utils-growpart
+ - dracut-network
+ # Bootloader
+ - grub2 grub2-efi ostree-grub2 efibootmgr shim
+ # SSH
+ - openssh-server openssh-clients
+ # Containers
+ - docker podman skopeo runc bubblewrap cri-o
+ - docker-lvm-plugin docker-novolume-plugin
+ - oci-umount
+ # Networking
+ - bridge-utils nfs-utils biosdevname iptables-services
+ - NetworkManager dnsmasq
+ # Storage
+ - lvm2 iscsi-initiator-utils sg3_utils
+ - device-mapper-multipath
+ - xfsprogs e2fsprogs mdadm
+ - cryptsetup
+ # Remote storage
+ - ceph-common glusterfs-fuse
+ # Time sync
+ - chrony
+ # Extra runtime
+ - authconfig sssd shadow-utils kexec-tools
+ # tuned
+ - tuned tuned-profiles-atomic
+ # Used by admins interactively
+ - sudo coreutils less tar xz gzip bzip2 rsync tmux
+ - nmap-ncat net-tools bind-utils
+ # Editors
+ - vim-minimal nano
+ # OpenShift
+ - origin-node origin-hyperkube origin-clients
```

Comment:
This list is missing `redhat-release-coreos`, no?

Reply:
Yes, good catch!  For reference here's a diff vs the base packages:

```
--- packages-old.txt	2018-05-23 15:32:18.246115057 -0400
+++ packages-new.txt	2018-05-23 15:32:43.910079944 -0400
@@ -1,8 +1,8 @@
 authconfig
-bash-completion
 bind-utils
 biosdevname
 bridge-utils
+bubblewrap
 bzip2
 ceph-common
 chrony
@@ -19,8 +19,6 @@
 dracut-network
 e2fsprogs
 efibootmgr
-filesystem
-glibc
 glusterfs-fuse
 grub2
 grub2-efi
@@ -32,7 +30,6 @@
 kernel
 kexec-tools
 less
-logrotate
 lvm2
 mdadm
 microcode_ctl
@@ -49,7 +46,7 @@
 origin-hyperkube
 origin-node
 ostree-grub2
-passwd
+podman
 policycoreutils-python
 rpm-ostree
 rsync
@@ -65,7 +62,6 @@
 subscription-manager-plugin-container
 subscription-manager-plugin-ostree
 sudo
-systemd
 tar
 tmux
 tuned
```

I squashed in most of the changes, although I dropped explicitly listing `filesystem glibc` because we really only want them via deps.

Reply:
That diff looks good to me!